### PR TITLE
fix(coinmarket): testnets support for trading

### LINKED ISSUE #14136: Coinmarket - cannot continue to buy Testnet BTC (+ icon missing)
### LOW PRIORITY (BUT NEEDS TO BE FIXED)
**Describe the bug**
Some providers are allowing to buy or sell testnet BTC. It is very limited amounts and only after test agreement with them.

**How to reproduce**

1. Go to trade section
1. Chose Testnet BTC (see screenshot 1) to buy with 50 EUR
1. Wait till offers load and then choose to compare all offers
1. See all offers with missing icon (see screenshot 2)
2. Select any offer
3. See disabled input (and receive address with validation error message) (see screenshot 3)

**Expected behavior**

**Screenshots**
1. 
image
2.
image
3.
image

### PR DESCRIPTION
**Pull request by FreeWall**
## Description

Add some testnet (tBTC, tXRP) support for trading

## Related Issue

Resolve #14136

### Before
image

### After
image

### CODE REVIEW

File: suite-common/wallet-config/src/networksConfig.ts [hunk 1]
```diff
@@ -479,7 +479,7 @@ export const networks = {
             },
         },
         coingeckoId: undefined,
-        coingeckoNativeId: undefined,
+        coingeckoNativeId: 'test-bitcoin', // fake, coingecko does not have testnets
```
Comment (tomasklim) [inline suite-common/wallet-config/src/networksConfig.ts:474]: Is it possible to upload our own bitcoin/ethereum/... testnet icons to `data.trezor.io`? If not, please uppercase the placeholder letter
Screenshot 2024-09-30 at 11 23 32

**Review by tomasklim — CHANGES_REQUESTED** [review]
1. Please hide `()` in case fiat is missing
Screenshot 2024-09-30 at 11 25 52

2. `TEST-BITCOIN` is there because it is not rebased right?
Screenshot 2024-09-30 at 11 28 29

File: suite-common/wallet-config/src/networksConfig.ts (hunk 1 above)
Comment (FreeWall) [reply suite-common/wallet-config/src/networksConfig.ts:474]: 1. fixed in https://github.com/trezor/trezor-suite/pull/14217/commits/d0e09693587bc4c82b1963aab2759159fff11686
2. fixed in https://github.com/trezor/trezor-suite/pull/14217/commits/d0e09693587bc4c82b1963aab2759159fff11686
3. it is possible to upload custom icons, I would create a separate issue for it; for now I changed the placeholder to uppercase
https://github.com/trezor/trezor-suite/issues/14656

**Review by tomasklim — APPROVED** [review]

**Pull request merged**